.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+- [ ] Refer to the issue that supports this Pull Request.
+- [ ] If the issue has missing info, explain the purpose/use case/pain/need that covers this Pull Request.
+- [ ] I've read the [CONTRIBUTING guide](https://raw.githubusercontent.com/conan-io/conan/develop/.github/CONTRIBUTING.md).
+- [ ] I've followed the PEP8 style guides for Python code.
+- [ ] I've opened another PR in the Conan docs repo documenting this one. https://github.com/conan-io/docs
```

Comment:
Add it to the changelog too? Maybe say here that a PR to the "develop" branch in the docs? The docs should have their CONTRIBUTING, but that would be too much, I suggest keeping the docs without formalisms at the moment.